libindex/libindex.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/quay/claircore/rhel/rhcc"
 	"github.com/quay/claircore/rpm"
 	"github.com/quay/claircore/ruby"
+	"github.com/quay/claircore/whiteout"
 )
 
 const versionMagic = "libindex number: 2\n"
```

Comment:
unrelated: just curious what this is for and why it's more detailed than just a number

Reply:
No idea about this one

Reply:
It gets thrown in to the `state` hash when `libindex`'s logic changes, to ensure that consumers can see that they should re-index manifests.